neondatabase/website
# feat: add sm size for subscribe form

This pull request aims to update design of the subscribe form within the blog post template.

Steps to check:

- open [preview](https://neon-next-git-article-subscribe-form-neondatabase.vercel.app/blog/json-features-pg-16)
- make sure that everything looks and feels as expected
